visier/connector/sessions.py

```diff
@@ -98,7 +100,7 @@ class VisierSession:
     def __init__(self, auth: Authentication) -> None:
         self._auth = auth
         self._session = None
-        self._timeout = 30
+        self._timeout = 300
```

Comment:
Is there a reason why this was increased by 10? Or was it a typo before?

Reply:
I had lowered it before to test OAuth callback expiry and forgot to change it back to 5min. 😞

Reply:
No, that should be lowered back. It was increased to support backend debugging